Runtime/Scripts/glTFast.asmdef

```diff
@@ -11,7 +11,13 @@
         "Unity.Meshopt.Decompress",
         "Unity.RenderPipelines.Universal.Runtime",
         "Unity.RenderPipelines.HighDefinition.Runtime",
-        "Unity.InputSystem"
+        "Unity.InputSystem",
+        "Unity.XR.ARFoundation",
+        "Unity.XR.ARCore",
+        "Google.XR.ARCoreExtensions",
+        "Unity.XR.ARSubsystems",
+        "Google.XR.ARCoreExtensions.GeospatialCreator",
+        "Unity.XR.CoreUtils"
```

Comment:
You might want to add these new dependencies to the [package.json](https://docs.unity3d.com/Manual/upm-manifestPkg.html) manifest as well.